src/signals/incident/definitions/wizard-step-2-vulaan/afval.ts

```diff
@@ -52,6 +52,7 @@ export const controls = {
       label: 'Kies de container waar het om gaat',
       shortLabel: 'Container(s)',
       pathMerge: 'extra_properties',
+      postPropertyNames: ['id', 'type', 'description'],
```

Comment:
Something like `allowedPropertyNames` would me more descriptive (same below in the utility function).

Reply:
When should properties be added to the `additional-questions.md` docs?